youtube_dl/extractor/ora.py

```diff
@@ -6,13 +6,14 @@
 from ..compat import compat_urlparse
 from ..utils import (
     get_element_by_attribute,
+    js_to_json,
     qualities,
     unescapeHTML,
 )
 
 
 class OraTVIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?ora\.tv/([^/]+/)*(?P<id>[^/\?#]+)'
+    _VALID_URL = r'https?://(?:www\.)?(ora\.tv|unsafespeech\.com)/([^/]+/)*(?P<id>[^/\?#]+)'
```

Comment:
Capturing groups increases memory usage. It also brings extra fields to the result of ```re.match(self._VALID_URL, url).groups()```. In some extractors the correctness relies on the number of capturing groups, so unnecassary groups should not be captured. Now it's a convention in youtube-dl: don't use capturing groups unless they're necessary.